app/models/setting.model.js

```diff
@@ -0,0 +1,15 @@
+module.exports = (sequelize, Sequelize) => {
+    const Setting = sequelize.define("setting", {
```

Comment:
I'm confused what a Setting will be used to represent, if it only has an `id` and `name` ... shouldn't there be a `name` and `value` fields, or something like that?

Reply:
I imagine it like this... 
When the story is generated in the LLM, it will take in key value pairs for parameters. In this case "Setting" is just a parameter to the story input, so when a setting is chosen, the parameter is send to the generator like this:
`
{
   "key":"setting",
   "value":"Enchanted Forest"
},
{
   "key":"language",
   "value":"English"
},
....
`

and so on...
So the "Setting" is the key of the parameter, to which the values will be stored in the dB.